app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
+        if type(config['enabled']) == str:
```

Comment:
I do wanna get this in ASAP, but given how much will depend on this I don't wanna let it linger more than necessary